multibuildingdetector/readers/xmldataset.py

```diff
@@ -36,10 +36,7 @@ def get_example(self, i):
         Returns:
             tuple of an image and bounding boxes and label
         """
-        if self._split == 'train':
-            id_ = self._img_ids[i * 2]
-        else:
-            id_ = self._img_ids[i * 2 + 1]
+        id_ = self._img_ids[i]
```

Comment:
I think due to this change the `__len__` attribute is not correct anymore. Probably @shorschig knows more about why this is needed.

Reply:
This is not needed. This was an attempt to split the dataset inside itself, using the same dataset but returning different images depending on whether it is used as training or test dataset.